protocol/contact.go

```diff
@@ -401,34 +401,27 @@ func (c *Contact) processSyncContactRequestState(remoteState ContactRequestState
 
 func (c *Contact) MarshalJSON() ([]byte, error) {
 	type Alias Contact
-	item := struct {
+	type ContactType struct {
 		*Alias
-		CompressedKey       string              `json:"compressedKey"`
 		Added               bool                `json:"added"`
 		ContactRequestState ContactRequestState `json:"contactRequestState"`
 		HasAddedUs          bool                `json:"hasAddedUs"`
 		Mutual              bool                `json:"mutual"`
-
-		Active        bool   `json:"active"`
-		PrimaryName   string `json:"primaryName"`
-		SecondaryName string `json:"secondaryName,omitempty"`
-	}{
-		Alias: (*Alias)(c),
+		Active              bool                `json:"active"`
```

Comment:
Since you're removing the `Active` field, it might be a good idea to double-check if it's not breaking the clients. In `status-mobile` I found this https://github.com/status-im/status-mobile/blob/develop/src/status_im2/contexts/contacts/events.cljs#L22

If it's a breaking change then it might be a good idea to create an issue in `status-mobile` and/or `status-desktop` so somebody (or you) can quickly work on the adaptations, otherwise most likely someone will upgrade `status-go` in the client and introduce a regression. I wonder if we're using semversion appropriately on the clients... :speak_no_evil: